crates/starknet-server/src/api/http/error.rs

```diff
@@ -18,23 +18,30 @@ pub enum HttpApiError {
     LoadError,
     #[error("The re-execution operation failed")]
     ReExecutionError,
+    #[error("The creation of empty block failed")]
+    CreateEmptyBlockError { msg: String },
+    #[error("The set time operation failed")]
+    BlockSetTimeError { msg: String },
+    #[error("The increase time operation failed")]
+    BlockIncreaseTimeError { msg: String },
 }
 
 impl IntoResponse for HttpApiError {
     fn into_response(self) -> axum::response::Response {
         let (status, error_message) = match self {
-            HttpApiError::GeneralError => {
-                (StatusCode::INTERNAL_SERVER_ERROR, String::from("general error"))
-            }
-            HttpApiError::FileNotFound => {
-                (StatusCode::BAD_REQUEST, String::from("file does not exist"))
-            }
+            err @ HttpApiError::GeneralError => (StatusCode::BAD_REQUEST, err.to_string()),
+            err @ HttpApiError::FileNotFound => (StatusCode::BAD_REQUEST, err.to_string()),
             err @ HttpApiError::DumpError { msg: _ } => (StatusCode::BAD_REQUEST, err.to_string()),
-            HttpApiError::LoadError => {
-                (StatusCode::BAD_REQUEST, String::from("load operation failed"))
+            err @ HttpApiError::LoadError => (StatusCode::BAD_REQUEST, err.to_string()),
+            err @ HttpApiError::ReExecutionError => (StatusCode::BAD_REQUEST, err.to_string()),
+            err @ HttpApiError::CreateEmptyBlockError { msg: _ } => {
```

Comment:
the `msg` property of the error is not used anywhere, only during assignment. Now the user will receive the string in the #[error(....)] attribute. Possible uses of the `msg` field: adding to the response, logging with warn! or error!.